kmir/src/kmir/__main__.py

```diff
@@ -41,12 +43,13 @@ def exec_run(
     input_file: str,
     definition_dir: str,
     output: str = 'none',
+    bug_report: bool = False,
     ignore_return_code: bool = False,
     **kwargs: Any,
 ) -> None:
     krun_output = KRunOutput[output.upper()]
-
-    kmir = KMIR(definition_dir, definition_dir)
+    br = BugReport(Path(input_file).with_suffix('.bug_report')) if bug_report else None
```

Comment:
@Baltoli I looked at the `KEVM` implementation for guidance, and the bug reports are being produced. But they are always produced at `{filename}.tar` not `{filename}.bug-report.tar` which I thought this line would of done. Before I figure out why, I just want to double check that `.bug_report` is preferred to included.

Reply:
Awesome, thank you very much!